app/react/Users/specs/Login.spec.js

```diff
@@ -81,12 +82,17 @@ describe('Login', () => {
     });
 
     describe('on response success', () => {
-      it('should reload thesauris and go to home', async () => {
+      it('should reload thesauris, set filters to include "restricted", and go to home', async () => {
```

Comment:
should this be in a separated unit test? even tho is the same action ?

Reply:
Since the title of the test is "on response success", I thought it would be enough to combine them.  Do you really think it merits a new test?